packages/suite/src/storage/migrations/index.ts

```diff
@@ -574,4 +575,24 @@ export const migrate: OnUpgradeFunc<SuiteDBSchema> = async (
             return account;
         });
     }
+
+    if (oldVersion < 38) {
+        await updateAll(transaction, 'devices', device => {
+            const { features } = device;
+            if (!features.internal_model) {
+                let deviceInternalModel;
+                switch (features.model.toUpperCase()) {
+                    case 'T':
+                        deviceInternalModel = DeviceModelInternal.T2T1;
+                        break;
+                    case '1':
+                        deviceInternalModel = DeviceModelInternal.T1B1;
+                        break;
+                    // no default
+                }
+                device.features.internal_model = deviceInternalModel;
```

Comment:
expect ts error here

Reply:
I can't see it 🤷 
![image](https://github.com/trezor/trezor-suite/assets/3729633/e0cd73c7-2ae2-4d8d-bfed-8729da32c77b)